hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmMetadataManagerImpl.java

```diff
@@ -742,6 +751,11 @@ protected void initializeOmTables(boolean addCacheMetrics)
     checkTableStatus(snapshotRenamedTable, SNAPSHOT_RENAMED_TABLE,
         addCacheMetrics);
     // TODO: [SNAPSHOT] Initialize table lock for snapshotRenamedTable.
+
+    compactionLogTable = this.store.getTable(COMPACTION_LOG_TABLE,
+        String.class, String.class);
```

Comment:
Shouldn't this be?
```suggestion
        String.class, CompactionLogEntry.class);
```

Reply:
Yes, copy-paste error. Thanks for catching.